spec/services/shopify_voider_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+require_relative '../../lib/active_merchant/billing/gateways/shopify.rb'
+
+describe ShopifyVoider do
+  let(:transaction_id) { '0xDEADBEEF' }
+  let(:order_id) { 'order_id' }
+  let(:transaction_amount) { 1 }
+  let(:transaction) { double(:transaction, amount: transaction_amount, id: transaction_id) }
+  let(:refunder_instance) { double('refunder_instance', perform: true) }
+
+  before do
+    allow(ShopifyAPI::Transaction).to receive(:find).and_return(transaction)
+  end
+
+  subject { described_class.new(transaction_id, order_id) }
+
+  context '.initialize' do
+    it 'successfully does it\'s thing' do
+      expect(subject).to be_truthy
```

Comment:
I would agree that this would get surfaced in other tests, but most of the time I write that spec then I write my initializer so it helps me validate that it works.